.github/workflows/update_docker_ci.yml

```diff
@@ -1,4 +1,5 @@
 name: Update CI docker image
+
```

Comment:
Had to change this file to actually trigger the build, fixed the issue here: https://github.com/XRPLF/clio/pull/2044

Reply:
Merged it, so it will work better now